lib/src/components/sprite_animation_component.dart

```diff
@@ -14,18 +14,24 @@ class SpriteAnimationComponent extends PositionComponent {
   bool removeOnFinish = false;
 
   /// Creates a component with an empty animation which can be set later
-  SpriteAnimationComponent();
+  SpriteAnimationComponent({
+    Vector2 position,
+    Vector2 size,
+    this.animation,
+    this.overridePaint,
+    this.removeOnFinish = false,
+  }) : super(position: position, size: size);
 
   /// Creates an [SpriteAnimationComponent] from an [animation] and a [size]
   ///
   /// Optionally [removeOnFinish] can be set to true to have this component be auto removed from the [BaseGame] when the animation is finished.
+  @Deprecated('Use SpriteAnimationComponent instead')
```

Comment:
On our current phase of the project (improving the APIs to release v1) it is ok to do breaking changes, so this constructor can be removed

Reply:
rip it off!

Reply:
This is not resolved.